setup.py

```diff
@@ -24,7 +24,9 @@
         'nmigen-yosys',
         'edalize==0.2.3',
         'fusesoc==1.11.0',
-        'nmigen @ git+https://github.com/nmigen/nmigen',
+        'nmigen @ git+https://github.com/nmigen/nmigen@'
+        'db5a981f438df7ec4f7f5b2224b5f01a36d79a44',
+        'wasmtime',
```

Comment:
Should we specify the version here? Maybe we could copy the version from nmigen-yosys?

Reply:
Right now the version is specified exactly like in nmigen-yosys.